mubench.reviewsite/tests/selenium/IndexTemplateTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Facebook\WebDriver\WebDriverBy;
+
+require_once 'WebDriverTestCase.php';
+
+class IndexTemplateTest extends WebDriverTestCase
```

Comment:
I don't like that these are called `XTemplateTest`. They're not specifically testing the templates, but the entire review site through the UI, which happens to be generated from a template. I think we should try to make these real scenario-based tests, like `ReviewFindingsTest`, `ManageRunsTest`, `ViewDetectorRunsTest`, ... probably most of the tests then happen to involve mainly a single template, but that doesn't define them.